app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -116,14 +116,14 @@ class ImageRegionSelectionInteractionViewTest {
           .setListener(onClickableAreaClickedListener)
       }
       onView(withId(R.id.clickable_image_view)).perform(
-        clickPoint(0.3f, 0.3f)
+        clickPoint(pointX = 0.3f, pointY = 0.3f)
       )
 
       verify(onClickableAreaClickedListener)
         .onClickableAreaTouched(
           capture(regionClickedEvent)
         )
-      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent("Region 3"))
+      assertThat(regionClickedEvent.value).isEqualTo(NamedRegionClickedEvent(regionLabel = "Region 3")) // ktlint-disable max-line-length
```

Comment:
Please don't use `// ktlint-disable max-line-length`. you can simply break this line into multiple lines. 

```suggestion
      assertThat(regionClickedEvent.value).isEqualTo(
        NamedRegionClickedEvent(
          regionLabel = "Region 3"
        )
      )
```

Reply:
Done!